css/main.css

```diff
@@ -165,9 +165,23 @@ body {
     letter-spacing: 0.7rem;
 }
 
+.fa {
```

Comment:
Really love the hover effect.

It did seem to increase the size (height) of each individual cell of a card. I personally preferred the more compact cards like before but I like this too, perhaps an updated theme (which I plan to work on later) would make it look better.